app/views/manuals/_header.html.erb

```diff
@@ -1,5 +1,8 @@
-<header aria-labelledby="manual-title" class="<%= presented_manual.hmrc? ? 'hmrc' : nil %>">
-  <div class='primary'>
+<header
+  aria-labelledby="manual-title"
+  class="<%= presented_manual.hmrc? ? 'hmrc' : nil %> <%= local_assigns.has_key?(:new_navigation) ? 'new-navigation' : ''%>">
```

Comment:
(NB: Please read comment on `show.html+new_navigation.erb` before addressing this issue)